webapp/src/Service/DOMJudgeService.php

```diff
@@ -1485,14 +1485,19 @@ public function getCompileConfig(Submission $submission): string
     /**
      * @return array<string, string>
      */
-    public function getVerdicts(bool $mergeExternal = false): array
+    public function getVerdicts(array $groups = ['final']): array
```

Comment:
If we create a (string backed?) enum for the groups, we can change it here.

Then we can add

```php
/**
 * @param VerdictGroup[] $groups
 */
```